source/integrate-with-integration-environment/process-identity-information.html.md.erb

```diff
@@ -0,0 +1,295 @@
+    },
+    {
+      "uprn": "10002345923",
+      "buildingName": "SAWLEY MARINA",
+      "streetName": "INGWORTH ROAD",
+      "dependentAddressLocality": "LONG EATON",
+      "addressLocality": "NOTTINGHAM",
+      "postalCode": "BH12 1JY",
+      "addressCountry": "GB",
+      "validUntil": "2022-01-01"
+    }
+  ],
+  "https://vocab.account.gov.uk/v1/passport": [
+     {
+        "documentNumber": "1223456",
+        "expiryDate": "2022-02-02"
+      }
+    ]
+}
+```
+## Understand your user’s core identity claim
+The `https://vocab.account.gov.uk/v1/coreIdentityJWT` property in the `/userinfo` response is the core identity claim, which is a JWT representing core identity attributes.
+
+The following are core identity attributes:
+
+* your user's name
+* your user’s date of birth
+* the level of identity confidence GOV.UK Sign In has reached
+
+<%= warning_text("If the <code>https://vocab.account.gov.uk/v1/coreIdentityJWT</code> property is not present, then GOV.UK Sign In was not able to prove your user's identity.") %>
+
+Our support team will give you the public key you must use to verify this JWT. We recommend you [use a certified JWT/JWS implementation](https://openid.net/developers/jwt/).
+
+The JWT contains the following claims:
+
+```
+{
+  "sub": "urn:fdc:gov.uk:2022:56P4CMsGh_02YOlWpd8PAOI-2sVlB2nsNU7mcLZYhYw=",
+  "iss": "https://identity.integration.account.gov.uk/",
+  "nbf": 1541493724,
+  "iat": 1541493724,
+  "exp": 1573029723,
+  "vot": "P2",
+  "vtm": "https://oidc.integration.account.gov.uk/trustmark",
+  "vc": {
+    "type": [
+      "VerifiableCredential",
+      "VerifiableIdentityCredential"
+    ],
+    "credentialSubject": {
+      "name": [
+        {
+          "validFrom": "2020-03-01",
+          "nameParts": [
+            {
+              "value": "Alice",
+              "type": "GivenName"
+            },
+            {
+              "value": "Jane",
+              "type": "GivenName"
+            },
+            {
+              "value": "Laura",
+              "type": "GivenName"
+            },
+            {
+              "value": "Doe",
+              "type": "FamilyName"
+            }
+          ]
+        },
+        {
+          "validUntil": "2020-03-01",
+          "nameParts": [
+            {
+              "value": "Alice",
+              "type": "GivenName"
+            },
+            {
+              "value": "Jane",
+              "type": "GivenName"
+            },
+            {
+              "value": "Laura",
+              "type": "GivenName"
+            },
+            {
+              "value": "O’Donnell",
+              "type": "FamilyName"
+            }
+          ]
+        }
+      ],
+      "birthDate": [
+        {
+          "value": "1970-01-01"
+        }
+      ]
+    }
+  }
+}
+```
+
+The `vc` claim in the JWT is a [verifiable credential (VC)](https://www.w3.org/TR/vc-data-model/). Claims about your user are contained in the `credentialSubject` JSON object.
```

Comment:
wondering if this is the right place for this piece of info - could get swallowed after such a long code example. Would there be a different place we could put this?

Reply:
sounds good